sdk/openai/openai/samples/cookbook/speechToSpeechChat/webpack.config.js

```diff
@@ -0,0 +1,53 @@
+// Generated using webpack-cli https://github.com/webpack/webpack-cli
```

Comment:
Is it standard for us to use WebPack here?  We've used rollup at least internally for the Azure SDK.

Reply:
For external-facing samples, I'd recommend webpack, since it's the more popular lib. Totally fair for internal-oriented examples.